code/socialDistribution/templates/tagtemplates/post.html

```diff
@@ -56,4 +57,8 @@ <h4 class="card-title mx-auto">{{ post.title }}</h4>
         {% csrf_token %}
         {% modal id="delete_post_modal" type="delete" label="Delete Post" title="Delete Post" btn="Delete"%}
     </form>
+    <form method="POST" action="{% url 'socialDistribution:editPost' post.id %}">
```

Comment:
I think we will have to send POST to `//service/author/{AUTHOR_ID}/posts/{POST_ID` from the spec.
Abram asked us not to create a different API endpoint. Please see here: 
https://eclass.srv.ualberta.ca/mod/forum/discuss.php?d=1836743

Sorry for the inconvenience. Did you want to do this in a new pr or on this one?